src/main/java/edu/kis/powp/jobs2d/drivers/SelectDriverMenuOptionListener.java

```diff
@@ -19,5 +19,9 @@ public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverMa
 	public void actionPerformed(ActionEvent e) {
 		driverManager.setCurrentDriver(driver);
 		DriverFeature.updateDriverInfo();
```

Comment:
must be removed (this line should be moved to subscriber instance)
unnecessary comments below